.coveragerc

```
@@ -639,6 +639,7 @@ omit =
     homeassistant/components/linode/*
     homeassistant/components/linux_battery/sensor.py
     homeassistant/components/lirc/*
+    homeassistant/components/livisi/climate.py
```

I think the following files should also be added here. It seems only `config_flow` has tests.
```suggestion
    homeassistant/components/livisi/__init__.py
    homeassistant/components/livisi/climate.py
    homeassistant/components/livisi/coordinator.py
    homeassistant/components/livisi/switch.py
```

---

I added this change, thank you.